build.gradle

```diff
@@ -43,7 +42,7 @@ repositories {
 }
 
 allprojects {
-  group 'org.opensearch.plugin'
+  group 'org.opensearch'
```

Comment:
Should be under plugin.

Reply:
no, it should be non-plugin for all project, and plugin for only the plugin zip.

Reply:
I see what you mean https://github.com/opensearch-project/custom-codecs/pull/27/files#diff-49a96e7eea8a94af862798a45174e6ac43eb4f8b4bd40759b5da63ba31ec3ef7R77

Reply:
Wait a sec, for plugins we publish `-client` artifacts and they are all under `org.opensearch.plugin`, for example:

https://mvnrepository.com/artifact/org.opensearch.plugin/ingest-attachment/2.9.0

Reply:
For info, looked with @peterzhuamazon : for non-core plugins, JAR artifacts go under `org.opensearch` but ZIP artifacts go under  `org.opensearch.plugin`. For core plugins, everything goes under ` org.opensearch.plugin`